src/Stryker.Core/Stryker.Core/Reporters/Progress/ConsoleOneLineLogger.cs

```diff
@@ -33,23 +33,31 @@ public void StartLog(string text, params object[] args)
 
         public void ReplaceLog(string text, params object[] args)
         {
-            int currentCursorTop = 0;
-            int currentCursorLeft = 0;
+            var currentCursorTop = 0;
+            var currentCursorLeft = 0;
             if (_hasConsole)
             {
                 currentCursorTop = Console.CursorTop;
                 currentCursorLeft = Console.CursorLeft;
 
                 Console.SetCursorPosition(0, _cursorTop);
                 ClearLine();
-
                 Console.SetCursorPosition(0, _cursorTop);
             }
 
             _logger.LogInformation(text, args);
 
             if (_hasConsole)
             {
+                if (Console.BufferWidth > 0)
```

Comment:
Is this a fix for #519 ?

Reply:
No. it turns out that Console.BufferWidth returns 0 on Linux to mark it as not supported.
The overall logic is to fix a race condition leading to a crash when you resize the console window